src/test/java/io/xstefank/wildfly/bot/PREditTest.java

```diff
@@ -38,14 +38,17 @@ void setUp() {
 
     @Test
     void configFileNullTest() throws IOException {
```

Comment:
This test can be deleted once the #118 is merged

Reply:
I agree, I just wanted to use multi line string instead of previous concatenation.